### PR TITLE
Added link to meeting notes for both GC and TC

### PR DESCRIPTION
**Pull request by kateetak**
This is to address #489.
Sergey had some inputs on that about adding a link to the video as well, I've added a comment to the issue about that.

### CODE REVIEW

**Review by bhs — APPROVED** [review]
Thanks!

File: README.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ a friendly, collaborative group and look forward to working together!
 * [Governance Committee (GC)](./governance-charter.md)
 * [Technical Committee (TC)](./tech-committee-charter.md)
 
+Both committes meet regularly and the respective meeting notes are publicly available in the [GC meeting notes](https://docs.google.com/document/d/1-23Sf7-xZK3OL5Ogv2pK0NP9YotlSa0PKU9bvvtQwp8) and the [TC meeting notes](https://docs.google.com/document/d/17v2RMZlJZkgoPYHZhIFTVdDqQMIAH8kzo8Sl2kP3cbY) Google Doc.
```
Comment (arminru) [inline README.md:35]:
```suggestion
Both committes meet regularly and the respective meeting notes are publicly available in the [GC meeting notes](https://docs.google.com/document/d/1-23Sf7-xZK3OL5Ogv2pK0NP9YotlSa0PKU9bvvtQwp8) and the [TC meeting notes](https://docs.google.com/document/d/17v2RMZlJZkgoPYHZhIFTVdDqQMIAH8kzo8Sl2kP3cbY) Google Docs.
```

**Review by arminru — APPROVED** [review]

**Comment by kateetak** [comment]
Thanks for updating and merging this.

**Comment by SergeyKanzhelev** [comment]
> Thanks for updating and merging this.

Thank you!

**Pull request merged**